Renaming IValue List functions

zdevito opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#32093 Renaming IValue List functions**

toGenericListRef -> toListRef
isGenericList -> isList
toGenericList -> toList
toXListRef -> toXVector

Differential Revision: [D19369767](https://our.internmc.facebook.com/intern/diff/D19369767)

zdevito committed:
Renaming IValue List functions

toGenericListRef -> toListRef
isGenericList -> isList
toGenericList -> toList
toXListRef -> toXVector

[ghstack-poisoned]

zdevito committed:
Update on "Renaming IValue List functions"


toGenericListRef -> toListRef
isGenericList -> isList
toGenericList -> toList
toXListRef -> toXVector

[ghstack-poisoned]

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit 2c88e2e6:


**None of the build failures appear to be your fault**.

* **1/1** recognized as flaky :snowflake:
    * Re-run these jobs?


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=2c88e2e6e6e7cf598ddbbb588edca503aefa8bc7).

### :snowflake: 1 failure recognized as flaky
The following build failures have been detected as flaky and **may not be your fault**:
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4217965) pytorch_linux_xenial_cuda9_cudnn7_py3_test (1/1)
**Step:** "Test" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=84127801) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=84127801)) :snowflake:
<details>
<summary>
<code>Jan 13 09:16:48 AssertionError: 11 not less than or equal to 1e-05 :</code>
</summary>

```
Jan 13 09:16:48 ---------------------------------------------------------------------- 
Jan 13 09:16:48 Traceback (most recent call last): 
Jan 13 09:16:48   File "/var/lib/jenkins/workspace/test/common_distributed.py", line 130, in wrapper 
Jan 13 09:16:48     self._join_processes(fn) 
Jan 13 09:16:48   File "/var/lib/jenkins/workspace/test/common_distributed.py", line 211, in _join_processes 
Jan 13 09:16:48     self._check_return_codes(elapsed_time) 
Jan 13 09:16:48   File "/var/lib/jenkins/workspace/test/common_distributed.py", line 231, in _check_return_codes 
Jan 13 09:16:48     self.assertEqual(p.exitcode, first_process.exitcode) 
Jan 13 09:16:48   File "/var/lib/jenkins/workspace/test/common_utils.py", line 890, in assertEqual 
Jan 13 09:16:48     super(TestCase, self).assertLessEqual(abs(x - y), prec, message) 
Jan 13 09:16:48 AssertionError: 11 not less than or equal to 1e-05 :  
Jan 13 09:16:48  
Jan 13 09:16:48 ---------------------------------------------------------------------- 
Jan 13 09:16:48 Ran 117 tests in 154.029s 
Jan 13 09:16:48  
Jan 13 09:16:48 FAILED (failures=1, skipped=7) 
Jan 13 09:16:48  
Jan 13 09:16:48 Generating XML reports... 
Jan 13 09:16:49 Traceback (most recent call last): 
Jan 13 09:16:49   File "test/run_test.py", line 456, in <module> 
Jan 13 09:16:49     main() 
```

</details>



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

suo reviewed (approved):
yay!

eellison commented:
Do you want to land this before or after tuple constant PR ? Tuple constant pr is a pretty big stack...

zdevito committed:
Update on "Renaming IValue List functions"


toGenericListRef -> toListRef
isGenericList -> isList
toGenericList -> toList
toXListRef -> toXVector

Differential Revision: [D19369767](https://our.internmc.facebook.com/intern/diff/D19369767)

[ghstack-poisoned]

zdevito committed:
Update on "Renaming IValue List functions"


toGenericListRef -> toListRef
isGenericList -> isList
toGenericList -> toList
toXListRef -> toXVector

Differential Revision: [D19369767](https://our.internmc.facebook.com/intern/diff/D19369767)

[ghstack-poisoned]
